manifest.php

```diff
@@ -23,7 +23,7 @@
     'label' => 'Job Scheduler',
     'description' => 'TAO job scheduler',
     'license' => 'GPL-2.0',
-    'version' => '0.9.0',
+    'version' => '2.1.0',
```

Comment:
are u sure?

Reply:
Rechecked twice. `0.9.0` it was my local version. After merging dev with version `2.0.1` i bumped it to `2.1.0`.